vsintegration/src/FSharp.Editor/Navigation/FindUsagesService.fs

```diff
@@ -23,22 +23,23 @@ type internal FSharpFindUsagesService
         projectInfoManager: ProjectInfoManager
     ) =
     
-    // File can be included in more than one project, hence single `range` may results with multiple `Document`s.
-    let rangeToDocumentSpans (solution: Solution, range: range, cancellationToken: CancellationToken) =
+    // File can be included in more than one project.
+    let rangeToDocumentSpan (solution: Solution, range: range, cancellationToken: CancellationToken) =
         async {
             if range.Start = range.End then return []
             else 
                 let! spans =
                     solution.GetDocumentIdsWithFilePath(range.FileName)
                     |> Seq.map (fun documentId ->
-                        async {
+                        asyncMaybe {
                             let doc = solution.GetDocument(documentId)
                             let! sourceText = doc.GetTextAsync(cancellationToken)
                             match CommonRoslynHelpers.TryFSharpRangeToTextSpan(sourceText, range) with
                             | Some span ->
                                 let span = CommonHelpers.fixupSpan(sourceText, span)
-                                return Some (DocumentSpan(doc, span))
-                            | None -> return None
+                                return DocumentSpan(doc, span)
```

Comment:
@smoothdeveloper 
It makes it tricky to code review, when other returns return None.  It becomes a thing you have to "know".